<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="CampoFecha">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hora:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampoHora">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampoNumAcceso">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Número de acceso:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampoNomPaciente">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Paciente:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampoNomDocRef">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del paciente:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampoIDPaciente">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre doctor de referencia:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampoEstudio">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Género:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampoGenero">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampoFechaNacimiento">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha de nacimiento:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Estudio:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Número de serie:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampoNumSerie">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CampoImagen">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Suelta una imagen aquí.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BotonGuardar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
